src/GetView.js

```diff
@@ -40,65 +34,43 @@ export default class GetView extends PureComponent {
       },
     };
 
+    const incidents = [];
+    let offset = 0;
     let response;
-
-    try {
+    do {
+      /* eslint-disable no-await-in-loop */
```

Comment:
This is a really bad practice, please have a look [here](https://eslint.org/docs/rules/no-await-in-loop) and refactor to use `Promise.all()`.

Reply:
I know, but in cases like this (when we need data from the response to end loop) this is fine as far as I know. If there is any other solution, please let me know.